challenges/linked-list/test_linked_list.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from linked_list import linked_list as LL
+
+
+@pytest.fixture
+def empty_ll():
+    return LL()
+
+
+@pytest.fixture
+def small_ll():
+    return LL([1, 2, 3, 4])
```

Comment:
Since these fixtures are already defined in the `conftest.py` file, you do not need to redefine them here.